packages/kit/src/core/sync/write_types/test/actions/+page.server.js

```diff
@@ -35,3 +35,53 @@ export const actions = {
 		return fail(400);
 	}
 };
+
+/** @type {import('./.svelte-kit/types/src/core/sync/write_types/test/actions/$types').SubmitFunction} */
```

Comment:
this could use a comment explaining what it does. it looks unused to me and none of the tests fail when it's removed, so I question what it's for